src/main/play-doc/operation/ClusterConfiguration.md

```diff
@@ -0,0 +1,180 @@
+# Configuring ConductR Cluster
```

Comment:
Other headlines such as `Restarting ConductR Cluster` or `Stopping ConductR Cluster` do not contain an `a` as well. So I'll stick with `Configuring ConductR Cluster`.